harpy/phase.py

```
@@ -8,9 +8,9 @@
 import rich_click as click
 from ._conda import create_conda_recipes
 from ._launch import launch_snakemake
-from ._misc import fetch_rule, fetch_report, snakemake_log
+from ._misc import fetch_rule, fetch_report, snakemake_log, ContigList
 from ._parsers import parse_alignment_inputs
-from ._validations import check_fasta, vcf_samplematch, validate_bam_RG, validate_input_by_ext
+from ._validations import check_fasta, vcf_sample_match, validate_bam_RG, validate_input_by_ext, fasta_contig_match, vcf_contig_match
```

Ruff

`._validations.fasta_contig_match` imported but unused

Remove unused import: `._validations.fasta_contig_match`

(F401)